braille_canvas.py

```diff
@@ -11,6 +12,40 @@
 from algutils.utils import EPSILON
 
 
+class Point(typing.NamedTuple):
+    x: float
+    y: float
```

Comment:
```suggestion
    y: float
    x: float
```
unless we change all remaining code to use x, y coordinates instead of y, x (which is a good idea).

Also y first, x second in `rect()` below.